crossdock/docker-compose.yml

```diff
@@ -76,6 +73,10 @@ services:
         image: jaegertracing/test-driver
         ports:
             - "8080"
+        links:
```

Comment:
U think it's even better to use `depends_on` container should be started in a separate network and reachable by name.